crates/primitives/src/kzg/generated.rs

```diff
@@ -0,0 +1,24 @@
+// @generated by build.rs from "src/kzg/trusted_setup.txt", do not modify manually.
+
+pub use c_kzg::{BYTES_PER_G1_POINT, BYTES_PER_G2_POINT};
+
+// Ensure that the build script constants are synced with the C bindings ones.
+const _: [(); BYTES_PER_G1_POINT] = [(); 48];
+const _: [(); BYTES_PER_G2_POINT] = [(); 96];
+
+pub const NUM_G1_POINTS: usize = 4096;
+pub const NUM_G2_POINTS: usize = 65;
```

Comment:
this is unfortunately not pub yet in kzg, so we need to define it, I'll open a PR to make the const public

Reply:
Ref to this issue: https://github.com/bluealloy/revm/issues/686 we can make change there